.gitignore

```diff
@@ -1,4 +1,9 @@
+venv/
+build/
+*.charm
 .tox/
+.coverage
 __pycache__/
-*.pyc
-*.charm
\ No newline at end of file
+*.py[cod]
+.idea
+.vscode/
```

Comment:
I err on the style to not put editors in a .gitignore -- but to add thees to the global gitignore

What do you think?

```suggestion
```

Reply:
Yeah, I agree. I think `charmcraft init` includes these, so I'll remove them.